File: programming essentials/access_inheritance/src/Circle.cpp
#include "Circle.h"

Circle::Circle(float r): Geoshape(r){}

void Circle::setRadius(float r)
{
    dim1 = dim2 = r;
}

float Circle::getRadius()
{
    return dim1;
}

float Circle::calculateArea()
{
    return 22.0/7.0 * dim1 * dim1;
}

File: Java/lab4/shape_class/Circle.java
public class Circle extends Shape {
    private Integer radius;

    public Integer getRadius() {
        return radius;
    }

    public void setRadius(Integer r) {
        radius = r;
    }

    @Override
    public void draw() {
        System.out.println("This is a Cicle with radius = "+getRadius());
    }
}